phase1/aws-ec2-terraform/secure-v1.2-user-data.yaml

```diff
@@ -197,7 +197,8 @@ write_files:
 
       systemctl daemon-reload
 
-      docker -v || env DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated --force-yes  docker-engine
+      apt-key adv --recv-keys --keyserver keys.ubuntu.com F76221572C52609D
```

Comment:
Is this not the same key that we already have above?

https://github.com/obi1kenobi/kubernetes-anywhere/blob/7191292c35bba7c11685dda90b97ff38b564ac96/phase1/aws-ec2-terraform/secure-v1.2-user-data.yaml#L20-L50